runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -106,67 +108,31 @@ template <typename LoaderDomain, typename SpecificLoader> class BaseLoader
-  void loadBinaryArithmetic(const Operator *op, ir::Graph &subg);
-  void loadAddV2(const Operator *op, ir::Graph &subg);
-  void loadPack(const Operator *op, ir::Graph &subg);
-  void loadResizeBilinear(const Operator *op, ir::Graph &subg);
-  void loadSelect(const Operator *op, ir::Graph &subg);
-  void loadSquaredDifference(const Operator *op, ir::Graph &subg);
-  void loadTranspose(const Operator *op, ir::Graph &subg);
-  template <ir::operation::Reduce::ReduceType reduce_type>
-  void loadReduce(const Operator *op, ir::Graph &subg);
-  void loadReduceAll(const Operator *op, ir::Graph &subg);
-  void loadReverseV2(const Operator *op, ir::Graph &subg);
-  void loadPad(const Operator *op, ir::Graph &subg);
+  void loadBinaryArithmetic(const Operator *op, ir::Graph &subg,
+                            ir::operation::BinaryArithmetic::ArithmeticType op_type);
+  void loadReduce(const Operator *op, ir::Graph &subg,
+                  ir::operation::Reduce::ReduceType reduce_type);
   void loadElementwiseActivation(const Operator *op, ir::Graph &subg,
                                  ir::operation::ElementwiseActivation::Type op_type,
                                  float alpha = 0.f, float beta = 0.f);
-  template <ir::operation::ElementwiseBinary::ElementwiseBinaryType op_type>
-  void loadElementwiseBinary(const Operator *op, ir::Graph &subg);
-  void loadElementwiseUnary(const Operator *op, ir::Graph &subg,
-                            ir::operation::ElementwiseUnary::Type op_type);
-  void loadExpandDims(const Operator *op, ir::Graph &subg);
-  void loadGather(const Operator *op, ir::Graph &subg);
+  void loadElementwiseBinary(const Operator *op, ir::Graph &subg,
+                             ir::operation::ElementwiseBinary::ElementwiseBinaryType op_type);
+  const ir::operation::ElementwiseUnary *
+  loadElementwiseUnary(const Operator *op, ir::Graph &subg,
+                       ir::operation::ElementwiseUnary::Type op_type);
   void loadCustom(const Operator *op, ir::Graph &subg);
-  void loadSpaceToBatchND(const Operator *op, ir::Graph &subg);
-  void loadBatchMatMul(const Operator *op, ir::Graph &subg);
-  void loadBatchToSpaceND(const Operator *op, ir::Graph &subg);
-  void loadSqueeze(const Operator *op, ir::Graph &subg);
-  void loadPrelu(const Operator *op, ir::Graph &subg);
-  void loadSplit(const Operator *op, ir::Graph &subg);
-  void loadSplitV(const Operator *op, ir::Graph &subg);
-  void loadSlice(const Operator *op, ir::Graph &subg);
-  void loadStridedSlice(const Operator *op, ir::Graph &subg);
-  void loadUnpack(const Operator *op, ir::Graph &subg);
   void loadComparison(const Operator *op, ir::Graph &subg);
-  void loadEinsum(const Operator *op, ir::Graph &subg);
-  void loadOneHot(const Operator *op, ir::Graph &subg);
-  void loadShape(const Operator *op, ir::Graph &subg);
-  void loadIf(const Operator *op, ir::Graph &subg);
-  void loadWhile(const Operator *op, ir::Graph &subg);
+
+  // FIXME Remove these methods
```

Comment:
I think it is ok to have some methods to keep the `switch` smaller.

Reply:
I agree with @s-barannikov comment. I think `switch` must smaller and simple pattern as possible.

Reply:
I think these operation's IR should be changed some params to input that can be tensor of backend. After it, the duplicated codes can be removed in this method. I will change this commend from removing these methods to remove duplicated codes in these methods.